src/packages/ConfigurationBuilders.Azure.nupkg/README.txt

```diff
@@ -0,0 +1,20 @@
+﻿########################################################################################################################
+##                                                                                                                    ##
+##                                   Microsoft.Configuration.ConfigurationBuilders                                    ##
+##                                                                                                                    ##
+########################################################################################################################
+
+Hello there. Sorry to bother you. If this is the first time you are installing this configuration builder into your
```

Comment:
I wouldn't bring up this message like this.  If you do want to say something "sorry", I would be more specific on what the sorry is about, like sorry that sorry if this annoys you....

Reply:
What about adding a "Warning" is more appropriate in this case, what do you think?